tests/test_utils.py

```diff
@@ -332,3 +333,26 @@ def test_labels(labels, fnc, expect):
             assert getattr(label, fn)(arg) == expect
         else:
             assert getattr(label, fn)() == expect
+
+
+@pytest.mark.parametrize(('v1', 'v2', 'match', 'valid'), [
+    ('1.0.4', '1.0.4', True, True),
+    ('5.3', '5.3.0', True, True),
+    ('2', '2.0.0', True, True),
+    ('7.3.1', '4.5.9', False, True),
+    ('9', '3.2.1', False, True),
+    ('6.8', '6.8.1', False, True),
+    ('bacon', '6.6.6', False, False),
+])
+def test_sanitize_version(v1, v2, match, valid):
+    if valid:
+        if match:
```

Comment:
Not sure if there's a lot of value in the `match` attribute. This is a simple string comparison.